CMakeLists.txt

```diff
@@ -41,6 +41,13 @@ qt_add_executable(SilverDict
 	src/query/queryscreen.h
 	src/query/queryscreen.cc
 	src/query/queryscreen.ui
+	src/query/wordlistmodel.h
+	src/query/wordlistmodel.cc
+	src/query/querymodel.h
+        src/query/querymodel.cc
+        src/edit/editDictionary.h
```

Comment:
You sure the filename's right?

Reply:
> You sure the filename's right?

Yes. But if you think there might be an issue, I'll double-check to be certain.